go-app-ussd.js

```diff
@@ -66,12 +66,17 @@ go.app = function() {
         });
 
         self.states.add('states:select_language', function(name){
+            var language_previously_not_set = self.im.user.lang === null;
```

Comment:
Oh, I see -- this is to direct people to different states depending on whether they're a new user or whether they're changing their language. Cool.